Tests/IntegrationTests.Shared/TestRunner.cs

```diff
@@ -34,11 +34,7 @@ internal static void Run(string title, Stream outputStream)
 			var testExecutionContext = TestExecutionContext.CurrentContext;
 			testExecutionContext.WorkDirectory = Environment.CurrentDirectory;
 
-#if __IOS__
```

Comment:
I committed/reverted that same change a few times too 😸 We should update our CI at some point!